vector/src/main/res/values/strings.xml

```diff
@@ -591,6 +591,66 @@
     <string name="settings_app_info_link_summary">Show the application info in the system settings.</string>
 
     <string name="settings_notification_privacy">Notification privacy</string>
+    <string name="settings_notification_troubleshoot">Troubleshoot Notifications</string>
+    <string name="settings_troubleshoot_diagnostic">Troubleshoot Diagnostic</string>
```

Comment:
I'm not sure about this, and I need more context. It reads like a command "troubleshoot a thing", but diagnostic sounds like you're troubleshooting the diagnostic. It could be "troubleshooting diagnostics".

Reply:
'Troubleshoot Notifications' is displayed in the 'settings screen', on tap it will display the screen running the tests. 'Troubleshoot Diagnostic' is used in this screen in the test summary (overlay pannel with summary of current tests execution), i will change it to 'troubleshooting diagnostics', makes more sens